app/presenters/contacts_finder_presenter.rb

```diff
@@ -0,0 +1,101 @@
+class ContactsFinderPresenter
+
+  def initialize(contact_group)
+    @contact_group = contact_group
+    @organisation = contact_group.organisation
```

Comment:
Doesn't that mean that this will vary depending which contact_group it's called through?  I'm thinking of the case where we'd want to republish all the index content items (eg from a rake task).  Given there's one content-item per organisation, I think it would be clearer if this was generated starting from the org.  The `public_updated_at` method could then become:

```ruby
def public_updated_at
  organisation.contact_groups.map(&:updated_at).max
end
```